### PR TITLE
alpha version

### PR DESCRIPTION
**Pull request by Stan1slav123**
- [DEMO LINK](https://Stan1slav123.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Stan1slav123.github.io/layout_search-bar-airbnb/report/html_report/)

The mockup is bad, it differs from "~npm test", and it doesn't include "text-shadow:" styles!

### CODE REVIEW

**Comment by Stan1slav123** [comment]
The search icon is bad too, what can i do with this mockup?

**Review by etojeDenys — CHANGES_REQUESTED** [review]
your test report link doesn't work, try to fix that

File: src/index.html [hunk 1]
```diff
@@ -10,7 +10,46 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page__body">
+    <form
+      data-qa="big"
+      class="search-bar search-bar--big"
```
Comment (VitaliyBondarenko1982) [inline src/index.html:16]:
```suggestion
      class="search-bar"
```
pay attention on this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md). same for other elements
Screenshot from 2023-09-08 13-16-42

File: src/index.html [hunk 2]
```diff
@@ -10,7 +10,46 @@
...
+      action="index.html"
+      method="get"
+    >
+      <button type="submit" class="search-button search-button--big">
```
Comment (VitaliyBondarenko1982) [inline src/index.html:20]: no need to place button here. icon just decorative element

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
All tests failed.
Try to fix them.

Please, check comments and [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

if you have any questions - fell freee to ask in chat

**Pull request closed without merging**